index.html

```diff
@@ -209,6 +209,16 @@ <h3 id="server-performance"><a href="#server-performance">Server performance</a>
             <td class="alert">no</td>
           </tr>
           <tr>
+            <td><a href="https://iojs.org/api/https.html">io.js</a></td>
+            <td class="ok">yes</td>
+            <td class="ok"><a href="https://github.com/joyent/node/pull/5882">yes</a></td>
+            <td class="ok"><a href="https://github.com/joyent/node/pull/7469">yes</a></td>
```

Comment:
How could OCSP stapling be enabled on client? :) Certainly it is. There are two events `OCSPRequest` - which is server-side, and `OCSPResponse` which is client-side.